themes/digital.gov/src/js/gh-edit-commit.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * Sets the most recent commit timestamp for the page
+ * This is displayed in #page-data section at the bottom of the page
+ */
+
+// eslint-disable-next-line func-names
+(function () {
+  const editPageLink = document.querySelector("#page-data .gh-commit-date");
+  const gitRepo = {
+    filepath: "",
+    branch: null,
+  };
+
+  /**
+   * Get hugo file path from the data-gh-edit-page attribute to link to Github repo location
+   * example string: news/2023/07/2023-07-19-gsa-shared-service-provider-program-guide.md
```

Comment:
You can use `@example` here.

```suggestion
   * @example
   * // returns string:
   * // news/2023/07/2023-07-19-gsa-shared-service-provider-program-guide.md
```

https://jsdoc.app/tags-example.html

Reply:
Ahh, thanks for the proper usage format.